pipelines/packageupmpublic.yaml

```diff
@@ -14,9 +20,25 @@ jobs:
   pool:
     vmImage: windows-2019
   steps:
+  - task: DownloadPipelineArtifact@2
+    displayName: Download tarballs
+    inputs:
+      source: specific
+      project: $(System.TeamProjectId)
+      ${{ if eq(parameters.SourcePipeline, 'stabilization_ci') }}:
+        pipeline: 32
+      ${{ if eq(parameters.SourcePipeline, 'mrtk_ci') }}:
+        pipeline: 15
+        runVersion: latestFromBranch
+        runBranch: refs/heads/releases/$(MRTKVersion)

```

Comment:
This might be a little overcomplicated. I'll update this to just download the branch that this pipeline is being run from.